src/main.rs

```diff
@@ -120,6 +127,10 @@ async fn main() {
         matches.is_present("skip_build"),
         matches.is_present("debug_build"),
```

Comment:
Separately, you could also have an enum for `BuildType`, something like:

```
enum Build {
    Skip,
    Debug,
    Release,
}
```

And then you could avoid a situation where someone incorrectly specifies "skip" and "debug"